aws-junit/pom.xml

```diff
@@ -58,7 +58,7 @@
     <dependency>
       <groupId>org.elasticmq</groupId>
       <artifactId>elasticmq-server_2.13</artifactId>
-      <version>1.5.6</version>
+      <version>1.5.7</version>
```

Comment:
only CVE trivy detects is a slightly out of date scala that this doesn't fix. however, not interested in overriding anything over a test lib not used elsewhere.